### ISSUE TITLE
A11y: element counts (m of n) are incorrect in dropdown in Chrome

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/calcite-components#2897]
### Summary

First pass through dropdown using keyboard provides wrong counts of items in group; subsequent passes while dropdown is open have correct counts.

### Actual Behavior

```
clickable  Open Dropdown  button 
Orange  radio menu item  checked  1 of 1
menu
Grape  radio menu item  not checked  2 of 2
Asparagus  not checked  1 of 1
Potato  checked  2 of 2
Yam  checked  3 of 3
Plant beans  1 of 1
Add peas  2 of 2
Apple  radio menu item  not checked  1 of 3
Orange  radio menu item  checked  2 of 3
Grape  radio menu item  not checked  3 of 3
Asparagus  not checked  1 of 3
Potato  checked  2 of 3
Yam  checked  3 of 3
Plant beans  1 of 2
Add peas  2 of 2
```

### Expected Behavior

```
clickable  Open Dropdown  button 
Orange  radio menu item  checked  2 of 3
menu
Grape  radio menu item  not checked  3 of 3
Asparagus  not checked  1 of 3
Potato  checked  2 of 3
Yam  checked  3 of 3
Plant beans  1 of 2
Add peas  2 of 2
Apple  radio menu item  not checked  1 of 3
Orange  radio menu item  checked  2 of 3
Grape  radio menu item  not checked  3 of 3
Asparagus  not checked  1 of 3
Potato  checked  2 of 3
Yam  checked  3 of 3
Plant beans  1 of 2
Add peas  2 of 2
```

### Reproduction Steps

Using example https://esri.github.io/calcite-components/?path=/story/components-buttons-dropdown--groups-and-selection-modes with NVDA on.

1. Tab to Open Dropdown and hit Enter (or click on Open Dropdown)
2. Arrow down through menu: counts are wrong
3. Continue arrow down with wraparound: counts are OK

If you close the dropdown and repeat the above steps--Tab variation only--without refreshing the page, you'll get the same results. A repeat using a click on the Open Dropdown may fail a time or two, but then works.

Tabbing down through menu has wrong counts; tabbing up has correct counts, except for the first group (one has to tab up to the first element to fix the first group's counts). (One cannot tab and have wraparound: after the last item in the menu, a tab closes the menu.)

### Relevant Info

Chrome 92

Firefox 91 is OK

**Comment by geospatialem** [comment]
The dropdown counts are coming in as-expected with NVDA in Chrome (`Version 103.0.5060.114`) and Firefox (`Version 91.11.0esr`). Can you verify, @MikeTschudi?

**Comment by MikeTschudi** [comment]
NVDA in Chrome (`Version 103.0.5060.114`) and Firefox (`Version 102.0.1`) works as expected with Calcite v1.0.0-next.516.  Thank you, everyone!
